lib/policy_machine_storage_adapters/active_record.rb

```diff
@@ -290,11 +278,14 @@ class Object < ObjectAttribute
     end
 
     class Operation < PolicyElement
-      has_and_belongs_to_many :policy_element_associations, class_name: 'PolicyMachineStorageAdapter::ActiveRecord::PolicyElementAssociation', join_table: 'operations_policy_element_associations'
```

Comment:
Yea, I tried that, but on a perf test on sandbox, one bulk privilege call for the read operation spiked the db to 50% utilization, so I took the line out.